src/gradletestrunnersmoketest.sh

```diff
@@ -4,9 +4,9 @@ gradletestrunnersmoketest () {
   export TEST_RUNNER_SERVER_SPECIFICATION_FILE="${NAMESPACEINUSE}.json"
   export TEST_RUNNER_DELEGATOR_SA_FILE="github-action-k8-sa.json"
   export TEST_RUNNER_SA_FILE="github-action-k8-sa.json"
-  cd ${GITHUB_WORKSPACE}/${workingDir}/datarepo-client
   echo "Building Data Repo client library"
-  ../gradlew clean assemble  
+  cd ${GITHUB_WORKSPACE}/${workingDir}
```

Comment:
Do we need to `cd` here?

Reply:
I don't think so, no. But I just wanted to be extra clear about which directory we're working in.